web/wp-content/themes/dctx-v2/template-sidebar-left.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-vc5tcg.php, but found phpcs-scan-vc5Tcg.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).